src/System.Memory/tests/Span/GcReporting.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.SpanTests
+{
+    public static class SpanGcReportingTests
```

Comment:
It am not sure whether it is worth duplicating between ReadOnlySpan and Span - it is pretty unlikely that we will have a bug in one but not in the other given the implementation . And this this is just a sanity test, not meant to be comprehensive.